ci/environment-dev-3.6.yml

```diff
@@ -51,7 +51,7 @@ dependencies:
   # Statistics
   - eofs
   - esmtools>=1.1.3
-  - xrft
+  - xrft=0.2.0
```

Comment:
Double `==` here as well.

Reply:
But it also works this way. Doesn't it? Travis does. I didn't find the problem with xrft. Something in power_spectrum for vwmp failed for the new release